site/src/components/DurationField/DurationField.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { useState } from "react";
+import { DurationField } from "./DurationField";
+
+const meta: Meta<typeof DurationField> = {
+  title: "components/DurationField",
+  component: DurationField,
+  args: {
+    label: "Duration",
+  },
+  render: function RenderComponent(args) {
+    const [value, setValue] = useState<number | undefined>(args.value);
+    return (
+      <DurationField
+        {...args}
+        value={value}
+        onChange={(value) => setValue(value)}
+      />
+    );
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof DurationField>;
+
+export const Hours: Story = {
+  args: {
+    value: hoursToMs(16),
+  },
+};
+
+export const Days: Story = {
+  args: {
+    value: daysToMs(2),
+  },
+};
```

Comment:
Since the value of `value` can be `undefined`, would we want a story for that, too?

Reply:
Makes sense!